### ISSUE TITLE
OpenDistro Index Management

### ISSUE THREAD

**Issue by Gustascern** [opendistro/for-elasticsearch#189]
Hello,
I'm trying to set up opendistro index management and for some reason i keep getting "index name [metricbeat-7.12.1-2022.03.08] does not match pattern '^.*-\\d+$'" error in console window. 
![159915023-758cb989-9731-4e95-95c4-18e0fdf01b11](https://user-images.githubusercontent.com/100773778/159915225-e64cc606-455d-4911-99f8-36ee47ffa258.png)

When I try to change the index to match the pattern, I get an error that you cant use those symbols (Invalid index name [filebeat-8.0.0-2022.02.28-/d], must not contain the following characters [ , ", *, \, <, |, ,, >, /, ?]")
![image](https://user-images.githubusercontent.com/100773778/159915381-fbe692c1-4548-4fe4-aab6-078b4e7f2c1e.png)
So basically these two terms are contradicting themselves, is there some sort of mistake or am i missing something. In the older Elasticsearch I have been using, the pattern is '^.*-\d+$', so without the double slashes, which accepts my pattern. Could it be that the slash was added by accident?
